src/oom/test-oomd-util.c

```diff
@@ -52,19 +53,20 @@ static void test_oomd_cgroup_kill(void) {
          * by the test so that pid1 doesn't delete it before we can read the xattrs. */
         cgroup = path_join(cgroup_root, "oomdkilltest");
         assert(cgroup);
+        assert_se(cg_create(SYSTEMD_CGROUP_CONTROLLER, cgroup) >= 0);
 
         /* If we don't have permissions to set xattrs we're likely in a userns or missing capabilities */
-        if (cg_set_xattr(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_test", "test", 4, 0) == -EPERM)
-                return (void) log_tests_skipped("no permissions to set user xattrs");
+        r = cg_set_xattr(SYSTEMD_CGROUP_CONTROLLER, cgroup, "user.oomd_test", "test", 4, 0);
+        if (IN_SET(r, -EPERM, -ENOTSUP))
```

Comment:
Maybe `ERRNO_IS_NOT_SUPPORTED(r)` ?

Reply:
Yes, please, let's use ERRNO_IS_NOT_SUPPORTED(r) in new code, please